pinakes/settings/defaults.py

```diff
@@ -413,3 +418,28 @@
 CORS_ALLOW_CREDENTIALS = False
 CSRF_TRUSTED_ORIGINS = env.list("PINAKES_CSRF_TRUSTED_ORIGINS", default=[])
 SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
+
+# Metrics collections
+PINAKES_INSIGHTS_TRACKING_STATE = env.bool(
+    "PINAKES_INSIGHTS_TRACKING_STATE", False
+)
+PINAKES_INSIGHTS_URL = env.str(
+    "PINAKES_INSIGHTS_URL", default="https//Your_insight_url"
```

Comment:
@hsong-rh Should the default be a valid production url that can be used out of the box. The only time it needs to change is if we are using staging env.

Reply:
I think the only issue if we do that is that if anyone is using it as a sandbox or testing environment, our production URL is going to end up with data that isn't meaningful and/or possibly get spammed

Reply:
@hsong-rh I understand that but asking a customer for a extra parameter doesn't make sense if we can use a default value that would work out of the box. The customer would just have to provide a userid/password and enable this option. I am talking to @Ladas if we can use mutual TLS and have the customers register their VM's or images then we don't even need the userid/password. The customer has to just enable the **PINAKES_INSIGHTS_TRACKING_STATE** a single config parameter and the rest works with all defaults

Reply:
When we did the upload on the cloud we used this
https://github.com/RedHatInsights/catalog_inventory-api/blob/5f88f98a51451302a18172df4c13358d53d84c88/deploy/clowdapp.yaml#L68

Reply:
@mkanoor PINAKES_INSIGHTS_URL is changed with valid link

Reply:
https://issues.redhat.com/browse/AA-1132

Reply:
@hsong-rh I have created a PR https://github.com/RedHatInsights/insights-analytics-collector/pull/7 if that gets merged then we can remove the username password variables and for catalog always use the certificates to do authentication.

Reply:
@hsong-rh for **testing purposes** on minikube/docker only you need to register the image with RHSM using the following command in your Dockerfile. 

**RUN subscription-manager register --username <<user>> --password <<password>>**

This will give us the certs and key needed to upload the payload to the insights